examples/maather_work_area/genzsprtwo-34.py

```diff
@@ -0,0 +1,25 @@
+import os
+_DIRECTORY = "./dir_organizer_files/"
+
+def directory_organizer(directory):
+    try:
+        contents = os.listdir(_DIRECTORY)
+        files = [content for content in contents if os.path.isfile(os.path.join(directory, content))]
+    except OSError as e:
+        print((f"error:{e}"))
+        return
+
+    for file in files:
+        if file.endswith(".txt"):
```

Comment:
We can directly get a list of files with a specified extension using the glob module.